dev-lang/rust/rust-1.55.0.ebuild

```diff
@@ -105,7 +105,11 @@ DEPEND="
 	net-misc/curl:=[http2,ssl]
 	sys-libs/zlib:=
 	dev-libs/openssl:0=
-	elibc_musl? ( sys-libs/libunwind:= )
+	elibc_musl? (
+		llvm-libunwind? ( sys-libs/llvm-libunwind:= )
+		!llvm-libunwind? ( sys-libs/libunwind:= )
+	)
+	libcxx? ( sys-libs/libcxx )
```

Comment:
... and `USE=libcxx` here look unnecessary.
Current policies are to not usually specify explicitly dependencies on C / C++ standard library, and `sys-libs/libcxx` is one of such implementations of C++ standard library.
Users should select libc++ as C++ standard library by `-stdlib=libc++` compiler option or `USE=default-libcxx` for `sys-devel/clang`.